src/models/base.es6.js

```diff
@@ -1,59 +1,41 @@
-import { EventEmitter } from 'events';
+const THING_ID_REGEX = new RegExp('t\\d_[0-9a-z]+', 'i');
 
 class Base {
-  constructor (props={}, opts={}) {
+  constructor (props={}) {
     this.props = {};
```

Comment:
Sure - probably a big enough change to be a separate PR. That would involve defining all properties for the data we expect to receive back from the API for each model, instead of cheating like this.